modules/core/src/main/scala/org/tessellation/domain/snapshot/SnapshotStorage.scala

```diff
@@ -0,0 +1,19 @@
+package org.tessellation.domain.snapshot
+
+import org.tessellation.schema.snapshot.Snapshot
+import org.tessellation.schema.transaction.Transaction
+import org.tessellation.schema.{Block, SnapshotOrdinal}
+import org.tessellation.security.hash.Hash
+import org.tessellation.security.signature.Signed
+
+trait SnapshotStorage[F[_], T <: Transaction, B <: Block[T], S <: Snapshot[T, B]] {
```

Comment:
what happened with `Global` prefix? 🤔

Reply:
We do not need `Global` prefix anymore. The storage works the same for `GlobalSnapshot` and `CurrencySnapshot` thus we can just parametrize it by `S`

Reply:
Eventually we can make a type alias for `Global` one

Reply:
ok, I'm fine with this :+1: